api/PaymentAddress.json

```diff
@@ -5,27 +5,13 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentAddress",
         "support": {
           "chrome": {
-            "version_added": "61",
-            "flags": [
-              {
-                "type": "preference",
-                "name": "#web-payments",
-                "value_to_set": "Enabled"
-              }
-            ]
+            "version_added": "60"
```

Comment:
Confirmed by https://mdn-bcd-collector.appspot.com/tests/api/PaymentAddress. Finding the commit for this is a bit tricky, but https://storage.googleapis.com/chromium-find-releases-static/259.html#259bce9e315438c00d7d660b446de65b6b03c360 happened soon after it was enabled and was M60, so it checks out.